docs/ocis/development/building-docs.md

```diff
@@ -26,9 +26,15 @@ hugo -D server
 
 Then open "http://localhost:1313/"
 
-When making changes to the docs, run `make docs` again and the server will pick up the changes and reload the page automatically
+When making changes to the docs, run 
+
+```bash
+rsync -ax --delete ../docs/ content/
+``` 
```

Comment:
needs to be `rsync -ax --delete ../docs/ content/ocis`

Reply:
no. run this in folder ocis/hugo to get all docs moved to hugo/contents.
docs/ocis, docs/extensions and docs/_index.md

Reply:
sorry, was confused. Was not aware that you are talking about the root level